templates/about.hbs

```diff
@@ -60,7 +61,7 @@
   </table>
 
   <p>You can still use <a href="https://crates.fyi/">crates.fyi</a> domain name for
-  redirects. <a href="https://crates.fyi/crates/clap"><code>https://crates.fyi/crates/clap</code></a> will
+  redirections. <a href="https://crates.fyi/crates/clap"><code>https://crates.fyi/crates/clap</code></a> will
   redirect to documentation of latest version of clap crate.</p>
```

Comment:
Ahh crates.fyi is actually working different.

Crates.fyi exists because, people can easily change `.io` with `.fyi` to access documentations. i.e: clap in crates.io: `crates.io/crates/clap`, replace .io with .fyi and it will redirect to `docs.rs/clap`.